tests/nonbonded/test_nonbonded_interaction_group.py

```diff
@@ -182,7 +182,7 @@ def ref_ixngroups(conf, params, box, lamb):
         conf,
         params,
         example_box,
-        lamb=lamb,
+        [lamb],
```

Comment:
Same here (this might be pretty significant; there are 4 lambda values)

Reply:
Resolved in [775576f](https://github.com/proteneer/timemachine/pull/699/commits/775576ffea4347658d94d4599ccd8d22817b36d5)